### PR TITLE
100- Add colour to global css root - Khadija

### LINKED ISSUE #100: Add colour to global css root
The colour E9FFE6 is used in all our designed pages but it is not in the global css root.
To add E9FFE6.

### PR DESCRIPTION
**Pull request by KhadijaHaji**
## Description

The global root CSS was missing a key colour (E9FFE6).
The colour was added to the root in global css. 

Fixes #100 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/styles/globals.css [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	--green13: #2f492b;
 	--green14: #63ad59;
 	--green15: #21c932;
-
+	--green15: #e9ffe6;
```
Comment (bazmurphy) [inline client/src/styles/globals.css]:
```suggestion
	--green16: #e9ffe6;
```

**Review by bazmurphy — CHANGES_REQUESTED** [review]
the numbers are the same 😄

File: client/src/styles/globals.css (hunk 1 above)
Comment (KhadijaHaji) [reply client/src/styles/globals.css]: thanks, all done!

**Review by bazmurphy — APPROVED** [review]

**Comment by bazmurphy** [comment]
the colour you added is now --green18 👍

**Pull request merged**